src/main/java/Task.java

```diff
@@ -0,0 +1,44 @@
+public class Task {
+    protected String taskDescription;
+    protected boolean isDone;
+
+    private static int numberOfTasks = 0;
+
+    public Task(String taskDescription) {
+        this.taskDescription = taskDescription;
+        this.isDone = false;
+        numberOfTasks += 1;
+    }
+
+    public String getTaskDescription() {
+        return taskDescription;
+    }
+
+    public void setTaskDescription(String taskDescription) {
+        this.taskDescription = taskDescription;
+    }
+
+    public static int getNumberOfTasks()
+    {
+        return numberOfTasks;
+    }
+
+    public void markAsDone()
+    {
+        isDone = true;
+    }
+
+    public void markAsUndone()
+    {
+        isDone = false;
+    }
+
+    @Override
+    public String toString() {
+        if (isDone) {
+            return "[X] " + taskDescription;
+        } else {
+            return "[ ] " + taskDescription;
```

Comment:
Perhaps you can consider making the checkbox strings into constants. The rationale is that this could potentially be a magic literal issue and code reviewers who are seeing the project for the first time may not easily infer the purpose of the string.